airqo-platform/AirQo-frontend
# Ft device status summary frontend

**What does this PR do?**
The PR is an improvement to the cards on `netmanager` dashboard following changes recommended by the SM
**What JIRA task is related to this PR?**
Not added in the JIRA board for this sprint.
**How do I test out this PR?**
Git clone this repo `cd AirQo-frontend/locate`. Then run `npm install` to get all the necessary dependencies. Afterwards, run `npm run local` to test out the system. Also, ensure that `device monitoring` backend microservice is running synchronously
